yarn-project/noir-contracts/src/contracts/slow_tree_contract/src/main.nr

```diff
@@ -0,0 +1,180 @@
+// This contract allow us to "read" public storage in private through a delayed tree.
+// More documentation need to be outlined for this properly, but there is some in 
+// https://github.com/AztecProtocol/aztec-packages/pull/2732/files
+// This is made as a separate contract for one thing mainly. Making it a simpler to use.
+contract SlowTree {
+    use dep::std::option::Option;
+    use dep::std::merkle::compute_merkle_root;
+    use dep::value_note::{
+        balance_utils,
+        utils::{increment, decrement},
+        value_note::{VALUE_NOTE_LEN, ValueNote, ValueNoteMethods},
+    };
+    use dep::aztec::{
+        context::{PrivateContext, PublicContext, Context},
+        note::{
+            note_header::NoteHeader,
+            utils as note_utils,
+        },
+        selector::compute_selector,
+        state_vars::{map::Map, public_state::PublicState, set::Set},
+        types::type_serialization::field_serialization::{
+            FieldSerializationMethods, FIELD_SERIALIZED_LEN,
+        },
+    };
+    use dep::slow_updates_tree::slow_map::{
+        SlowMap, Leaf, SlowUpdateProof
+    };
+
+    global TREE_HEIGHT: Field = 254;
+    global MEMBERSHIP_SIZE: Field = 256; // TREE_HEIGTH + 2
```

Comment:
```suggestion
    global MEMBERSHIP_SIZE: Field = 256; // TREE_HEIGHT + 2
```

Ser, did you not enable the spell check extension? You are facing a typo Karen here
<img width="381" alt="image" src="https://github.com/AztecProtocol/aztec-packages/assets/13470840/f461ad35-e390-49fe-b901-25ce8f4b5d31">